cmd/internal/base/base.go

```diff
@@ -16,7 +17,10 @@ import (
 	"text/template"
 	"unicode"
 
+	migrate2 "ariga.io/atlas/sql/migrate"
```

Comment:
is it really necessary?

Reply:
Yes. To create the `LocalDir`.

Reply:
I meant the package alias. You can give a different name to the `migrate` variable, but if you find it prettier/readable this way, keep it.